Validator/Constraints/ReCaptchaVerified.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace IDCI\Bundle\ExtraFormBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target("PROPERTY")
```

Comment:
...

Reply:
I think this force the constraint to be applied on properties and not on classes by example.

Reply:
This is the default behavior ?

Reply:
The default behavior allow to apply the constraint on class and on property.